src/main/webapp/nutrition.js

```diff
@@ -0,0 +1,71 @@
+//Use user input to search for possible foods
+function foodSearch() {
+    let userfood = document.getElementById('foodInput').value;
+    let url = 'https://trackapi.nutritionix.com/v2/search/instant?query=' + userfood;
+    let myHeaders = new Headers({ 
+        'x-app-id': '85f8ef5f',
+        'x-app-key': '7dac3e04e347a6dc70bfd89e404501d3'
```

Comment:
You should try to hide your api keys for various reasons. Here is one way to do that
https://gist.github.com/derzorngottes/3b57edc1f996dddcab25